examples/evm/nft-linker/ERC721Demo.sol

```diff
@@ -1,13 +1,49 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.0;
 
-import '@openzeppelin/contracts/token/ERC721/ERC721.sol';
+import "@openzeppelin/contracts/token/ERC721/ERC721.sol";
+import "@openzeppelin/contracts/token/ERC721/extensions/ERC721URIStorage.sol";
 
-//A simple ERC721 that allows users to mint NFTs as they please.
-contract ERC721Demo is ERC721 {
-    constructor(string memory name_, string memory symbol_) ERC721(name_, symbol_) {}
+/**
+ * @title ERC721 Demo
+ * @author Axelar
+ * @notice A simple ERC721 that allows users to mint NFTs as they please.
+ */
+contract ERC721Demo is ERC721URIStorage {
```

Comment:
Should we have `Demo` in the name here? We don't have it in other examples.

Reply:
I think this is what is called before, but ya don't see much use for the name will change this